includes/class-sensei-course.php

```diff
@@ -1090,31 +1090,40 @@ public function course_count( $post_status = 'publish' ) {
 	 * course_lessons function.
 	 *
 	 * @access public
-	 * @param int    $course_id (default: 0)
+	 * @param int    $course_id   (default: 0)
 	 * @param string $post_status (default: 'publish')
-	 * @param string $fields (default: 'all'). WP only allows 3 types, but we will limit it to only 'ids' or 'all'
+	 * @param string $fields      (default: 'all'). WP only allows 3 types, but we will limit it to only 'ids' or 'all'
+	 * @param array  $base_args   Base arguments for the WP query.
+	 *
 	 * @return array{ type WP_Post }  $posts_array
 	 */
-	public function course_lessons( $course_id = 0, $post_status = 'publish', $fields = 'all' ) {
+	public function course_lessons( $course_id = 0, $post_status = 'publish', $fields = 'all', $base_args = [] ) {
```

Comment:
I think this can be named `$query_args`, so it's a bit more familiar

Reply:
✅ d9a3158fdc5d85d55a9dc77700487098fc75b0a2